controllers/tasksRequests.js

```diff
@@ -169,45 +169,62 @@ const addOrUpdate = async (req, res) => {
   }
 };
 
-const approveTaskRequest = async (req, res) => {
+const updateTaskRequests = async (req, res) => {
   try {
     const { taskRequestId, user } = req.body;
     if (!taskRequestId) {
       return res.boom.badRequest("taskRequestId not provided");
     }
 
-    const response = await taskRequestsModel.approveTaskRequest(taskRequestId, user);
+    const { action = TASK_REQUEST_ACTIONS.APPROVE } = req.query;
```

Comment:
Are we APPROVing by default?

Reply:
<img width="979" alt="Screenshot 2023-11-30 at 4 23 21 AM" src="https://github.com/Real-Dev-Squad/website-backend/assets/98796547/05f67bbe-8569-427d-99b2-2c11ca3ee131">
As you can see in this screen shot same function is used for previous api which was only used to approve. Will remove this while removing the previous api code.